docs/settings.md

```diff
@@ -6,11 +6,6 @@ You can access **Settings** for Remix by clicking the gear icon on the left side
 
 ![Remix General Settings Page](images/settings/a-settings-general.png)
 
```

Comment:
Please leave this image the title - but remove the image on line 22 (because it contains the language switcher - which is soon to be removed).